src/c++/perf_analyzer/test_concurrency_manager.cc

```diff
@@ -313,7 +237,33 @@ TEST_CASE("concurrency_infer_type")
     params.streaming = false;
   }
 
+  const std::string json_str{R"(
```

Comment:
It isn't making sense to me that this is required for every test. The common case of the real code would not have this. Why is it needed for every test?

If, for some reason it is needed for every test, then it should be pulled into the test class constructor or member function

Reply:
still looking into how to refactor this

Reply:
If you remove all of this except for line 264 it works fine.